include/net/aws_fota.rst

```diff
@@ -64,6 +63,33 @@ The implementation uses a job document similar to the following (where *bucket_n
 
 The current implementation uses information from the ``host`` and ``path`` fields only.
 
+For upgrading MCUBoot you'll have to do a similar process as the above except you'll have to turn on `CONFIG_SECURE_BOOT` and `CONFIG_MCUBOOT_BUILD_S1_VARIANT` for your application this will result in two files being generated when building your application which ends in *s0/s1_image_update.bin* in the build folder of your application. The URL of these files are added to the path starting with the *s0* file followed by the *s1* file where the path is separated by a space.
+
+.. note::
+  It is important that you place the path of *s0* before *s1* in the job document.
+
+
+.. parsed-literal::
+   :class: highlight
+
+   {
+     "operation": "bootloader_fw_update",
```

Comment:
Is the operation value used anywhere? If not, we don't need all this information, but just a sentence stating that for a bootloader update, you need to specify "s0.bin s1.bin" instead of "file_name.bin" and link to more information.

Reply:
It's not used so it can be just removed. Where should i link to more information?

Reply:
Either here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/net/fota_download.html#lib-fota-download
Or here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_bootloader.html#upgradable-bootloader
The AWS FOTA library will always use the FOT download library, correct? So then we can use the first one (which also links to the second).

Reply:
For its current implementation yes. Can't tell the future but I would assume it will continue to use FOTA download library for a while.

Added reference to FOTA download library which describes how to enable bootloader upgrades. And removed the additonal JSON and tried to write a sentance about changing **file_name.bin** to **s0.bin s1.bin**